tests/test_url_finders.py

```diff
@@ -72,12 +70,13 @@ def test_get_only_links_from_css(tmp_path):
             'url_finders': {'text/css': 'get_css_links'},
         }
 
+        freeze_config['extra_pages'] += ["/static/style.css"]
         freeze(module.app, freeze_config)
 
     assert (builddir / 'index.html').exists()
     assert (builddir / 'static' / 'OFL.txt').exists()
-    assert not (builddir / 'static' / 'style.css').exists()
-    assert not (builddir / 'static' / 'TurretRoad-Regular.ttf').exists()
+    assert (builddir / 'static' / 'style.css').exists()
+    assert (builddir / 'static' / 'TurretRoad-Regular.ttf').exists()
```

Comment:
Now there's no page that didn't match, so the result is the same as if all links were followed...

Reply:
It means to update whole `app_links_css`

Reply:
Yes. (Or create a new app, but that would probably be more work.)